### ISSUE TITLE
If you die  just for  a few dozens of RMB,  then what’s the point?

### ISSUE THREAD

**Issue by Inseeker** [996icu/996.ICU#1656]
If you die  ,everything will make no sense.
Developers' lives matter.

Many young people died around the age of 30. 
What’s the point.

As a developer, if you just die for tens of YUAN an hour.  
Then people's lives is less important than a dog.
Do you feel bad?  Do you feel sad ?

**Comment by Inseeker** [comment]
The most important thing is to be healthy and alive

**Comment by LeonXtp** [comment]
> If you die ,everything will make no sense.
> Developers' lives matter.
> 
> Many young people died around the age of 30.
> What’s the point ?
> 
> As a developer, if you just die for tens of YUAN an hour.
> Then people's lives is less important than a dog.
> Do you feel bad? Do you feel sad ?

You're right, but living in China, too many coders don't have that much choices as most companies are doing the same, woking overtime or being eliminated, what do you choose?

**Comment by feixiaohuijava** [comment]
> > If you die ,everything will make no sense.
> > Developers' lives matter.
> > Many young people died around the age of 30.
> > What’s the point ?
> > As a developer, if you just die for tens of YUAN an hour.
> > Then people's lives is less important than a dog.
> > Do you feel bad? Do you feel sad ?
> 
> You're right, but living in China, too many coders don't have that much choices as most companies are doing the same, woking overtime or being eliminated, what do you choose?

@LeonXtp  u r right, there is no more reasonable choice under most circumstances in China

**Comment by Inseeker** [comment]
> > If you die ,everything will make no sense.
> > Developers' lives matter.
> > Many young people died around the age of 30.
> > What’s the point ?
> > As a developer, if you just die for tens of YUAN an hour.
> > Then people's lives is less important than a dog.
> > Do you feel bad? Do you feel sad ?
> 
> You're right, but living in China, too many coders don't have that much choices as most companies are doing the same, woking overtime or being eliminated, what do you choose?

I understand the situation what you said.  But life is so precious that there is nothing can be compared with, we should cherish it.

**Comment by Akatsuki-Ryu** [comment]
> > If you die ,everything will make no sense.
> > Developers' lives matter.
> > Many young people died around the age of 30.
> > What’s the point ?
> > As a developer, if you just die for tens of YUAN an hour.
> > Then people's lives is less important than a dog.
> > Do you feel bad? Do you feel sad ?
> 
> You're right, but living in China, too many coders don't have that much choices as most companies are doing the same, woking overtime or being eliminated, what do you choose?

this is such a sad story . I know what you feel . I had been living in china as well. yes . many people don't really have a choice . people who has a choice will choose an easy way out.. but those easy way out is not for everyone after all

**Comment by Akatsuki-Ryu** [comment]
> > > If you die ,everything will make no sense.
> > > Developers' lives matter.
> > > Many young people died around the age of 30.
> > > What’s the point ?
> > > As a developer, if you just die for tens of YUAN an hour.
> > > Then people's lives is less important than a dog.
> > > Do you feel bad? Do you feel sad ?
> > 
> > 
> > You're right, but living in China, too many coders don't have that much choices as most companies are doing the same, woking overtime or being eliminated, what do you choose?
> 
> I understand the situation what you said. But life is so precious that there is nothing can be compared with, we should cherish it.

yes . that is so right . but at some point , one have to choose even he doesn't like just for the sake of what he want . for example Money
